### ISSUE TITLE
Material You theming doesn't seem to work

### ISSUE THREAD

**Issue by NoahAndrews** [dslul/openboard#488]
**Describe the bug**
The button color doesn't match the rest of the wallpaper-derived system color

**Screenshots**
OpenBoard: ![Screenshot_20220116-165122.png](https://user-images.githubusercontent.com/10224994/149681499-7a461c98-9b3a-4a30-9674-f7b9389f75a5.png)

GBoard: ![Screenshot_20220116-165130.png](https://user-images.githubusercontent.com/10224994/149681507-eb162956-ee45-416d-8e47-09a08ca0c779.png)

**Smartphone (please complete the following information):**
 - Device: Pixel 6
 - OS: Android 12

**Comment by dslul** [comment]
What happens if you change the wallpaper to a red-ish one?

**Comment by NoahAndrews** [comment]
A reddish wallpaper actually does show a reddish button. I guess I just dislike how dark the tinting is.

**Comment by MajeurAndroid** [comment]
Yes, I agree with you. The problem is that currently blue colored UI elements are tinted to the system color, which is not the right way to do this. It would be better to apply the sysytem color directly **as** background color of a key for exemple. For tinted icons, we'll have to get them as vector drawables, only for api 31+, and color them directly through xml.
I leave some hints because I do not own an android 12 phone, so for now I will not be able to take care of this :)
